tests/automation/group_testing.spec.ts

```diff
@@ -143,8 +142,9 @@ test_group_Alice_1W_Bob_1W_Charlie_1W(
     await clickOnTestIdWithText(aliceWindow1, 'edit-group-name');
     await typeIntoInput(aliceWindow1, 'group-name-input', '     ');
     await aliceWindow1.keyboard.press('Enter');
-    const errorMessage = aliceWindow1.locator('.error-message');
-    await expect(errorMessage).toContainText('Please enter a group name');
+    await waitForMatchingText(aliceWindow1, 'Please enter a group name');
+    // const errorMessage = aliceWindow1.locator('.error-message');
+    // await expect(errorMessage).toContainText('Please enter a group name');
```

Comment:
to be removed?

Reply:
No, i've put adding in a data-test id into the backlog for an error message here, so i thought i would just leave it for future...